### PR TITLE
MAP-72-Refactor-skip-link

### PR DESCRIPTION
**Pull request by davidthomas22**

### CODE REVIEW

File: server/views/pages/home.njk [hunk 1]
```diff
@@ -11,62 +11,72 @@
     {% set isFemalePrison = true %}
 {% endif %}
```
Comment (davidthomas22) [inline server/views/pages/home.njk:13]: added `skipLink` block to override `main` block behaviour and ensure that the skip link takes you to the homepage content

File: server/views/partials/layout.njk [hunk 2]
```diff
@@ -66,6 +67,14 @@
     {% endblock %}
 </title>
 
+{% block skipLink %}
```
Comment (davidthomas22) [inline server/views/partials/layout.njk:70]: Added to layout partial to ensure skip link underline is enforced on page

File: server/views/pages/home.njk [hunk 3]
```diff
@@ -11,62 +11,72 @@
...
 
-{% block content %}
+{% block skipLink %}
```
Comment (davidthomas22) [inline server/views/pages/home.njk:14]: added `skipLink` block to override `main` block behaviour and ensure that the skip link takes you to the homepage content

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Review by GurnankCheema — CHANGES_REQUESTED** [review]
Code for skip link all okay but there's a small blip in the layout as grey footer now extends to  top of main content. probably just a missing div etc - now resolved

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**